sds_data_manager/batch/efs-access-batch/efs_check.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
I think you will need to explain to me the purpose of this code here. 

I'm also wondering if once the dust settles with Opensearch being replaced we might want to optimize our Dockerfiles so that we only have one for all Dockerized lambdas. But that would be for another PR.

Reply:
I will add some comment in the function about its purpose.